### ISSUE TITLE
N-trimmer might have an overflow on Right bp trim

### ISSUE THREAD

**Issue by msettles** [ibest/HTStream#215]
"Paired_end": {
        "in": 59766627,
        "out": 59766627,
        "discarded": 0,
        "Read1": {
            "basepairs_in": 2305056164,
            "basepairs_out": 2306023101,
            "leftTrim": 1887576,
            "rightTrim": 18446744073706697103. <-----
        },
        "Read2": {
            "basepairs_in": 6036429327,
            "basepairs_out": 6028476739,
            "leftTrim": 7335460,
            "rightTrim": 617128
        }

**Comment by joe-angell** [comment]
what version?

**Comment by msettles** [comment]
Should be in master, I'm on a different branch, but merged master this morning and have made no modifications to N-trimmer

**Comment by joe-angell** [comment]
Yeah I cannot reproduce this, any chance you could get the read in question and the parameters used for ntrimmer?
